Darkly/LDClient.m

```diff
@@ -53,6 +55,13 @@ - (BOOL)start:(LDConfigBuilder *)inputConfigBuilder userBuilder:(LDUserBuilder *
                 LDClientManager *clientManager = [LDClientManager sharedInstance];
                 [clientManager syncWithServerForConfig];
                 [clientManager startPolling];
+                
+                EventSource *source = [EventSource eventSourceWithURL:[NSURL URLWithString:kStreamUrl] apiKey:ldConfig.apiKey];
```

Comment:
When backgrounding is set up, will we need to specify that we're doing background execution?